app/src/main/java/com/portfolio/tasky/networking/usecases/domain/TaskyCallStatus.kt

```diff
@@ -0,0 +1,9 @@
+package com.portfolio.tasky.networking.usecases.domain
+
+interface TaskyCallStatus {
```

Comment:
This one either

Reply:
This is required since I want to start stop loading UI over here.